### ISSUE TITLE
Swap Usage calculation on Windows sometimes wrong

### ISSUE THREAD

**Issue by exa-mk** [it-novum/openitcockpit-agent-go#56]
**Agent Mode**:
* [ ] Pull Model (openITCOCKPIT Server request data from Agent)
* [x] Push Model (Agent send data to openITCOCKPIT)

**Versions**
* openITCOKPIT Server Version: 4.2.1
* openITCOCKPIT Monitoring Agent Version: 3.0.5

**Operating system**
Windows Server 2019 Datacenter 

* Do you use an HTTP proxy? no
* Are you using Auto-TLS, HTTPS or HTTP? https
* Used CPU architecture: amd64

**Describe the bug**
In some circumstances then swap usage is calculated wrong somehow:
```
    "swap": {
        "total": 1324351488,
        "percent": 149.32699920823435,
        "used": 1977614336,
        "free": 3641704448,
        "sin": 0,
        "sout": 0
    },
```
![image](https://user-images.githubusercontent.com/69202794/123260354-c14c2100-d4f5-11eb-86ec-aa95e23d550c.png)

There is more free than total and used together... ;)
(Usage 150% :D )

**To Reproduce**
Good question... no idea. It's a Windows Terminal Server with some web services. On our other Windows Servers it seems to be fine.

**Expected behavior**
Proper calculation; no misleading alarms.

**Comment by nook24** [comment]
This looks like an integer overflow. Windows uses an uint32 in Megabytes to report the current swap usage. We converted Megabytes into bytes but still using a uint32 which results in a maximum swap filesize of 4GB. I think this is the issue.

I will change this to an uint64

**Comment by nook24** [comment]
Fixed with 3.0.6: https://github.com/it-novum/openitcockpit-agent-go/releases/tag/3.0.6
